storybook/stories/AdaRedemptionDialog.stories.js

```diff
@@ -28,21 +29,34 @@ storiesOf('AdaRedemptionForm', module)
       <AdaRedemptionForm
         onSubmit={action('submit')}
         isSubmitting={false}
-        isRedemptionDisclaimerAccepted={true}
+        isRedemptionDisclaimerAccepted
         isCertificateSelected={false}
         isCertificateEncrypted={false}
         onCertificateSelected={action('onCertificateSelected')}
         onPassPhraseChanged={action('onPassPhraseChanged')}
         onRedemptionCodeChanged={action('onRedemptionCodeChanged')}
         onChooseRedemptionType={action('onChooseRedemptionType')}
-        redemptionCode=''
-        redemptionType='regular'
+        redemptionCode=""
+        redemptionType="regular"
         getSelectedWallet={() => ({})}
         wallets={[
           { value: 'wallet-1', label: 'First Wallet' },
           { value: 'wallet-2', label: 'Second Wallet' },
           { value: 'wallet-3', label: 'Third Wallet' },
         ]}
+        suggestedMnemonics={[]}
```

Comment:
@daniloprates please use `wordList` for suggestedMnemonics like here: https://github.com/input-output-hk/daedalus/blob/develop/storybook/stories/PaperWallets.stories.js#L59
Please do the same on other places as well. Thanks!

Reply:
Done! ☝🏽